### DISCUSSION TITLE
Sometimes the IDE Explorer does not list the files in the workspace even when they are there

### DISCUSSION THREAD

**Discussion by bayodesegun** [eclipse-theia/theia#12421] [General]
### Bug Description:

Sometimes the IDE Explorer does not list the files in the workspace even when they are there.

- Clicking the refresh button **does not work.**
- Reloading the page **does not work**.
- Reloading the frame **does not work**.
- If I right-click on the empty space and create a new file, then the files show up.

Any idea why this could be happening and what can be done to recover from the situation without user intervention? 

### Steps to Reproduce:

- Happens randomly.

![image](https://user-images.githubusercontent.com/15055526/232085938-57547073-8b7a-4560-86ab-fe15e7258f65.png)

### Additional Information

- Operating System: Linux. Theia is running on a Linux container.
- Theia Version: 1.35.0.
- IDE is loaded in an iframe.
- IDE is running behind an Nginx proxy.

The workspace opened in the image above is `/home/user`. It seems that the IDE can find the `user` directory (as indicated by the `EXPLORER: USER` text), but it does not list the contents of the `user` directory.

**Comment by JonasHelming** [comment]
Do you have an example workspace to reproduce this?

**Reply by bayodesegun** [reply]
Yes, this is used widely on [this app](https://app.theconstructsim.com), and happening randomly there.

**Reply by JonasHelming** [reply]
This seems like a hosted custom product. Are you developing it? If so, could it also be that there is a problem with the file system mounting? Reason I am asking is that to my knowledge, we have not observed this issue in any adopter project so far.

**Reply by bayodesegun** [reply]
Yes, it's the app is a custom product, and we are using the IDE there. And yes, I'm developing it. Maybe there is a problem with the file system mounting, but are there error messages we should see if such a problem occurs?

Whenever this happens, we find the files are indeed there, but are just not listed. The top-level workspace itself (`/home/user`) is always detected, but sometimes the files and directories inside it are not listed.

**Reply by JonasHelming** [reply]
Does this issues also occur if you run your product locally without the cloud set-up? It is a little tricky to give generic advise here without debugging your specific set-up tbh. The issue is at least unknown, meaning I have never experiences this in any other Theia-based product.
@vince-fugnitto Do you have any idea here?

**Reply by vince-fugnitto** [reply]
> @vince-fugnitto Do you have any idea here?

I don't unfortunately :(
